backend/src/utils/resourceUtils.ts

```diff
@@ -905,6 +910,66 @@ export const migrateTemplateDisablement = async (
     });
 };
 
+/**
+ * TODO: There should be a better way to go about this... but the namespace is unlikely to ever change
+ */
+export const isRHOAI = (fastify: KubeFastifyInstance): boolean =>
+  fastify.kube.namespace === 'redhat-ods-applications';
```

Comment:
DSC has a `release` attribute under status, and they detail in there wether we use ODH or RHOAI, it would be yet another request and I don't think the namespace would change in the future, but that's another possibility to determine the installation

Reply:
It does -- but it's custom text and requires a fetch to find out -- I'd rather not build in the logic to fetch it to do this... I did think of that. This is reliable if not awkward.

Reply:
Yeah, we haven't changed the namespace and I would assume if done, we should have a long list of requirements to change and that can fall into that bucket, I think this a reliable way to check the operator.